Frontend/lib/screens/address/address_list.dart

```diff
@@ -9,8 +12,23 @@ class AddressList extends StatefulWidget {
 }
 
 class _AddressListState extends State<AddressList> {
+  final titles = ["List 1", "List 2", "List 3"];
+  final subtitles = [
```

Comment:
what does this do?

Reply:
I will remove this, I was trying to hardcode for listing.

Reply:
oh okay